ci/bootstrap.py

```diff
@@ -232,8 +239,22 @@ async def run(self):
                 if j._mount_docker_socket:
                     mount_options.extend(['-v', '/var/run/docker.sock:/var/run/docker.sock'])
 
+                if j._unconfined:
+                    security_options = ['--security-opt', 'seccomp=unconfined', '--security-opt', 'apparmor=unconfined']
+                else:
+                    security_options = []
+
                 main_cid, main_ok = await docker_run(
-                    'docker', 'run', '-d', *env_options, *mount_options, j._image, *j._command
+                    'docker',
+                    'run',
+                    '-d',
+                    *env_options,
+                    *mount_options,
+                    *security_options,
```

Comment:
This change makes an assumption that you can always do `--entrypoint j._command[0] IMAGE j._command[1:]`. Does this work in all cases?

Reply:
This should work with single element command lists so I can't think of a case where it wouldn't work in all cases. That being said this is grabbed from Cotton's deadbeef changes so I can try without it.